lab-saul/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "lab-saul",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "lint": ".eslintrc",
```

Comment:
This script isn't working. Try to get it to run (you need to bring another module into your program to make this happen)